prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1531,6 +1534,16 @@ int efa_rdm_ope_post_remote_write(struct efa_rdm_ope *ope)
 		if (OFI_UNLIKELY(!pkt_entry))
 			return -FI_EAGAIN;
 
+		if (ope->fi_flags & FI_INJECT) {
+			assert(ope->iov_count == 1);
+			assert(ope->total_len <= ep->inject_size);
+			copied = ofi_copy_from_hmem_iov(pkt_entry->wiredata + sizeof(struct efa_rdm_rma_context_pkt),
+				ope->total_len, FI_HMEM_SYSTEM, 0, ope->iov, ope->iov_count, 0);
```

Comment:
This doesn't look correct, `FI_HMEM_SYSTEM` needs to be replaced by the real hmem iface.

Reply:
Sorry I am wrong here, it should be system as there is no MR passed in for inject